### ISSUE TITLE
False positive if svn.simple directory is empty

### ISSUE THREAD

**Issue by GoogleCodeExporter** [pentestmonkey/unix-privesc-check#1]
```
[UPC024] WARNING: Cleartext subversion passsword file: 
/root/.subversion/auth/svn.simple/*
```

Original issue reported on code.google.com by `pentestm...@googlemail.com` on 9 Nov 2010 at 4:53
